frontend/src/components/layout/Footer.scss

```
@@ -35,14 +42,29 @@
   }
 
   .links-column {
-    width: 145px;
-    @media (max-width: $mdBreakpoint) {
-      width: 200px;
+    width: 174px;
+    @media (max-width: 1052px) {
+      width: 164px;
+    }
+    @media (max-width: 992px) {
+      width: 152px;
+    }
+    @media (max-width: 950px) {
+      width: 130px;
+    }
+    @media (max-width: 900px) {
+      width: 300px;
+    }
+    @media (max-width: 900px) {
+      width: 300px;
+    }
```

```suggestion
    @media (max-width: 900px) {
      width: 300px;
    }
```

This one is duplicated.

Though breakpoints every 50px seems like a lot, all good if you think this is the best way but I wonder if we should use grid/fractions or percentage along with min and max width.

---

The issue I found with that is that we have 3 columns with dynamic size so in smaller interactions it would break the layout, so that's why I added this strict granularity.
(Also left a duplicate by mistake, that I'm removing ....)